idempotency/src/main/kotlin/com/ritense/idempotency/domain/IdempotencyEvent.kt

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2015-2023 Ritense BV, the Netherlands.
+ *
+ * Licensed under EUPL, Version 1.2 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://joinup.ec.europa.eu/collection/eupl/eupl-text-eupl-12
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" basis,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.ritense.idempotency.domain
+
+import java.time.LocalDateTime
+import java.util.UUID
+import javax.persistence.Column
+import javax.persistence.Entity
+import javax.persistence.Id
+import javax.persistence.Table
+
+@Entity
+@Table(name = "idempotency_event")
+class IdempotencyEvent(
+
+    @Id
+    @Column(name = "id", columnDefinition = "BINARY(16)", updatable = false)
+    val id: UUID,
+
+    @Column(name = "consumer", columnDefinition = "VARCHAR(150)", updatable = false)
+    val consumer: String,
+
+    @Column(name = "event_id", columnDefinition = "VARCHAR(150)", updatable = false)
```

Comment:
message_id imo.

Reply:
I disagree you send a message but process an event. Otherwise why do we have EventListeners?